docs/content/integrations/airflow.mdx

```diff
@@ -1,15 +1,11 @@
 ---
 title: "Airflow & Dagster | Dagster Docs"
-description: The dagster-airflow package allows you to import Airflow DAGs into Dagster jobs and assets, as well as trigger job runs from Airflow.
+description: Explore the options for migrating from Airflow to Dagster.
 ---
 
 # Airflow & Dagster
 
-<CodeReferenceLink filePath="examples/with_airflow" />
-
-The [`dagster-airflow`](/\_apidocs/libraries/dagster-airflow) library provides interoperability between Dagster and Airflow. This integration is designed to help support users who have existing Airflow usage and are interested in exploring Dagster.
-
-Using `dagster-airflow`, you can incorporate all or some of your existing Airflow Directed Acyclic Graphs (DAGs) or orchestrate Dagster jobs from Airflow. Refer to the [Airflow reference](/integrations/airflow/reference) for examples or the following sections for info on migrating from Airflow.
+Migrating from Airflow to Dagster, or integrating Dagster into your existing workflow orchestration stack can be accomplished in many ways. The <a href="/guides/migrations/migrating-airflow-to-dagster">Pick your own journey</a> guide provides a variety suggestions in how one can migrate their Airflow pipelines to Dagster, or build a platform where both tools co-exist.
```

Comment:
nit: I'd add a comma to ease readability

```suggestion
Migrating from Airflow to Dagster, or integrating Dagster into your existing workflow orchestration stack, can be accomplished in many ways. The <a href="/guides/migrations/migrating-airflow-to-dagster">Pick your own journey</a> guide provides a variety suggestions in how one can migrate their Airflow pipelines to Dagster, or build a platform where both tools co-exist.
```

Reply:
+1